pkg/commands/models/branch.go

```diff
@@ -68,3 +71,7 @@ func (b *Branch) HasCommitsToPull() bool {
 func (b *Branch) IsRealBranch() bool {
 	return b.Pushables != "" && b.Pullables != ""
 }
+
+func (b *Branch) IsDetachedHead() bool {
+	return b.DisplayName != ""
```

Comment:
I would rather add a new field called DetachedHead (bool) than depend on this display name because I can imagine the display name being populated for a branch that's not a detached head